### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by AdelGhabboun**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Adel Ghabboun
```
Comment (haukurstefansson) [inline LICENSE]: Just wanted to check if there are any legal issues related to this. Does this need to reference Microsoft? Like

` Copyright (c) Microsoft Corporation.`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-netapp-files
+description: "This project demonstrates how to create a Snapshot Policy for Microsoft.NetApp resource provider using .NETCore SDK.
+---
+
+# Azure NetAppFiles SDK Sample - Dual-Protocol for .NETCore
```
Comment (haukurstefansson) [inline README.md]: It says Dual-Protocol here, should probably be changed to Snapshot Policy

**Comment by haukurstefansson** [comment]
There are a bunch of extra files (.dll, .json, .sln, and more) included in this pull request that I don't think are supposed to be there

File: LICENSE (hunk 1 above)
Comment (AdelGhabboun) [reply LICENSE]: This was auto generated so I am not sure about it. I will look at the old samples and maybe @paulomarquesc can chime in

Comment (AdelGhabboun) [reply LICENSE]: you are right. changed it

**Pull request merged**